src/main/java/com/googleinterns/zoomtube/utils/TranscriptLineUtil.java

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.googleinterns.zoomtube.utils;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.googleinterns.zoomtube.data.TranscriptLine;
+import java.util.Date;
+import java.util.concurrent.TimeUnit;
+
+/** Provides methods to create TranscriptLine Entities and TranscriptLine objects. */
+public final class TranscriptLineUtil {
+  public static final String KIND = "TranscriptLine";
+  public static final String LECTURE = "lecture";
+  public static final String START = "start";
+  public static final String DURATION = "duration";
+  public static final String CONTENT = "content";
+  public static final String END = "end";
+
+  private TranscriptLineUtil() {}
```

Comment:
Please move this at then end because it is a useless method. Please see: The order you choose for the members and initializers of your class can have a great effect on learnability. However, there's no single correct recipe for how to do it; different classes may order their contents in different ways.

https://google.github.io/styleguide/javaguide.html#s3.4.2-ordering-class-contents